.github/workflows/unit_tests.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         # the torchaudio env var does nothing when torchaudio is installed, but doesn't require it's presence when it's not
         pip install lilcom '.[tests]'
         # Enable some optional tests
-        pip install h5py dill smart_open[http] kaldi_native_io webdataset==0.2.5 s3prl scipy nara_wpe pyloudnorm ${{ matrix.extra_deps }}
+        pip install h5py dill smart_open[http] kaldi_native_io webdataset==0.2.5 s3prl scipy nara_wpe pyloudnorm ${{ matrix.extra_deps }} multi-storage-client==0.16.0
```

Comment:
We'll have to move this requirement to `extra_deps` field in python version matrix, because Python 3.8 tests are failing (looks like MSC doesn't support that version anymore). Please add this to all python version tests starting from 3.9